content/docs/http/validation.md

```diff
@@ -264,13 +264,13 @@ Let's take an example of the `unique` validation rule. We want to ensure the use
 export const updateUserValidator = vine
   .compile(
     vine.object({
-      email: vine.string().unique((query, field) => {
-        query.whereNot(
-          'id',
-          // highlight-start
-          field.meta.userId
-          // highlight-end
-        )
+      email: vine.string().unique(async (db, value, field) => {
+        const user = await db
+          .from('users')
+          .whereNot('id', field.meta.userId)
```

Comment:
Hey, can we highlight the `whereNot('id')` clause using `// highlight-start` and `// highlight-end` fences?

Reply:
Yeah no problem i will do it :)